src/test/java/com/patientpal/backend/patient/service/PatientServiceTest.java

```diff
@@ -0,0 +1,187 @@
+
+            // then
+            assertNotNull(response);
+            assertThat(response.getMemberId()).isEqualTo(patient.getMember().getId());
+            assertThat(response.getNokContact()).isEqualTo(patient.getNokContact());
+        }
+
+        @Test
+        void 환자_프로필을_조회할_때_프로필이_없으면_예외가_발생한다() {
+            // given
+            when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.empty());
+
+            // when & then
+            assertThatThrownBy(() -> patientService.getProfile(patient.getMember().getUsername()))
+                    .isInstanceOf(EntityNotFoundException.class);
+        }
+
+        @Test
+        void 환자_프로필을_성공적으로_수정한다() {
+            // given
+            when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.of(patient));
+            PatientProfileUpdateRequest request = updatePatientProfileRequest();
+
+            // when
+            patientService.updatePatientProfile(patient.getMember().getUsername(), request);
+
+            // then
+            assertThat(patient.getPatientSignificant()).isEqualTo(UPDATE_PATIENT_SIGNIFICANT);
+            assertThat(patient.getNokContact()).isEqualTo(UPDATE_NOK_CONTACT);
+        }
+
+        @Test
+        void 환자_프로필을_수정할_때_프로필이_없으면_예외가_발생한다() {
+            // given
+            when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.empty());
+            PatientProfileUpdateRequest request = updatePatientProfileRequest();
+
+            // when & then
+            assertThatThrownBy(() -> patientService.updatePatientProfile(patient.getMember().getUsername(), request))
+                    .isInstanceOf(EntityNotFoundException.class);
+        }
+
+        @Test
+        void 환자_프로필을_성공적으로_삭제한다() {
+            // given
+            when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.of(patient));
+
+            // when
+            patientService.deletePatientProfile(patient.getMember().getUsername());
+
+            // then
+            verify(patientRepository).delete(patient);
+            Optional<Patient> deletedPatient = patientRepository.findById(patient.getId());
+            assertThat(deletedPatient).isEmpty();
+        }
+
+        //TODO - PENDING이 하나라도 있을 시 삭제 불가능
+//    @Test
+//    void 환자_프로필을_삭제할_때_진행중인_매칭이_있으면_예외가_발생한다() {
+//        // given
+//        when(patientRepository.findByMember(patient.getMember())).thenReturn(Optional.of(patient));
+//        // 진행 중인 매칭 존재 시 예외 발생하도록 설정
+//        doThrow(new IllegalStateException("진행 중인 매칭이 있어 프로필을 삭제할 수 없습니다."))
+//                .when(patientRepository).delete(any(Patient.class));
+//
+//        // when, then
+//        assertThatThrownBy(() -> patientService.deletePatientProfile(patient.getMember().getUsername()))
+//                .isInstanceOf(IllegalStateException.class)
+//                .hasMessage("진행 중인 매칭이 있어 프로필을 삭제할 수 없습니다.");
+//    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 158.